Gruntfile.js

```diff
@@ -74,6 +72,6 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-force-task');
   grunt.loadNpmTasks('grunt-run');
 
-  grunt.registerTask('test', [ 'ts', 'run:test', 'prettier', 'tslint:src', 'tslint:test']);
+  grunt.registerTask('test', [ 'run:compile', 'run:test', 'prettier', 'tslint:src', 'tslint:test']);
```

Comment:
this was causing differences in the map output between the `npm start` version and the ghprb version